gamemode/modules/base/sh_createitems.lua

```diff
@@ -297,8 +297,6 @@ local function addEntityCommands(tblEnt)
     local function defaultSpawn(ply, tr, tblE)
         local ent = ents.Create(tblE.ent)
         if not ent:IsValid() then error("Entity '" .. tblE.ent .. "' does not exist or is not valid.") end
-        ent.dt = ent.dt or {}
-        ent.dt.owning_ent = ply
```

Comment:
This code is suspicious. Not only am I calling the `Setowning_ent` function, I also set `dt` if it hasn't been already, *and* the dt var itself. I guess I should have documented why I did this shit, now I'm being paranoid about it working around some obscure bug where ent.dt is nil.

Reply:
It was compatibility with the old [DTVar method](https://wiki.garrysmod.com/page/Entity/DTVar), which is now an internal function to NetworkVar.

Even [GMod itself](https://github.com/Facepunch/garrysmod/commit/3e138636eb1b0ad6ed785dedf350020755cff5f1#r16402605) has broken compatibility for it in the past, so I'm not sure how widespread its usage is anymore.